From 4ee8d876b25a1345804b66213b92a69602abdd83 Mon Sep 17 00:00:00 2001
From: Henning Schild <henning.schild@siemens.com>
Date: Mon, 18 Sep 2017 18:02:10 +0200
Subject: [PATCH] ivshmem-net: slightly improve debug output

There where two lines with the same error message, change one of them.

Signed-off-by: Henning Schild <henning.schild@siemens.com>
Signed-off-by: Jan Kiszka <jan.kiszka@siemens.com>
(cherry picked from commit d66af388dd2512ea7c7a776c731409854ed40a45)
---
 drivers/net/ivshmem-net.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ivshmem-net.c
+++ b/drivers/net/ivshmem-net.c
@@ -369,7 +369,7 @@ static void ivshm_net_tx_clean(struct ne
 
 		data = ivshm_net_desc_data(in, &in->tx, desc, &len);
 		if (!data) {
-			netdev_err(ndev, "bad tx descriptor\n");
+			netdev_err(ndev, "bad tx descriptor, data == NULL\n");
 			break;
 		}
 
